cli/flox/src/commands/init/mod.rs

```diff
@@ -122,14 +122,13 @@ impl Init {
         };
 
         let env = if customization.packages.is_some() {
-            Dialog {
-                message: "Installing Flox suggested packages...",
-                help_message: None,
-                typed: Spinner::new(|| {
-                    PathEnvironment::init(PathPointer::new(env_name), &dir, &customization, &flox)
-                }),
-            }
-            .spin()?
+            info_span!(
+                "init_with_suggested_packages",
+                progress = "Installing Flox syggested packages"
```

Comment:
Install and uninstall are part of the sdk, the core of them being `CoreEnvironment::{install,uninstall}`
I think i took the `instrument` _off_ those methods tho because it created additional, somewhat redundant layers, where we had more usecasey messages on the cli side.